controlplane/hack/deploy/addon/work-manager/hub/crds/hive.openshift.io_clusterclaims.yaml

```diff
@@ -0,0 +1,163 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
```

Comment:
side topic, do we still require these crds after removing bma controller?

Reply:
no, will remove these crds after removing bma controller